api/hypershift/v1beta1/nodepool_types.go

```diff
@@ -730,6 +731,7 @@ type KubevirtNodePoolPlatform struct {
 	//
 	// +optional
 	// +kubebuilder:validation:Enum=Enable;Disable
+	// +kubebuilder:default=Enable
```

Comment:
how does this impact existing clusters?

Reply:
@qinqon wdyt?

Reply:
It all depends on what hypershift kubevirt provider is doing with VMs if NodePool definition change afte an upgrade since kubevirt VMs are inmutable, so it may mean that workers need to be restarted.